pkg/controller/certificaterequests/venafi/venafi.go

```diff
@@ -104,31 +112,51 @@ func (v *Venafi) Sign(ctx context.Context, cr *cmapi.CertificateRequest, issuerO
 		}
 	}
 
-	certPem, err := client.Sign(cr.Spec.CSRPEM, duration, customFields)
+	pickupID, exists := cr.ObjectMeta.Annotations[VenafiPickupIDAnnotation]
```

Comment:
nit: perhaps `hasPickupID` instead of `exists`?

Reply:
> We can just check if pickupID == "" from what I can tell?

well i am going with that one less variable :)